### ISSUE TITLE
Package name guesser breaks on modern Go with go.mod.

### ISSUE THREAD

**Issue by biztos** [biztos/binsanity#1]
_That's what you get for try'na be smart!_

## Old behavior:

If package name not provided, crawls up the source tree to find a likely candidate.

## Expected behavior:

Generates file with correct package declaration.

## New behavior, probably since Go 1.11 or 1.12

```bash
~/foo $ tree .
.
├── assets
│   └── foo.txt
├── bar
│   ├── assets
│   │   └── bbb.txt
│   └── boo.go
└── main.go

4 directories, 4 files
~/foo $ binsanity assets
~/foo $ grep package binsanity.go
package main
~/foo $ cd bar
~/foo/bar $ binsanity assets
Error finding import path: Path /Users/frost/foo/bar not found under src in $GOPATH /Users/frost/go
~/foo/bar $
```

__NOTE:__ same error occurs even if a proper `go.mod` file is present.

**Comment by biztos** [comment]
Fixed for 1.0.0

**Comment by biztos** [comment]
Fixed in #v1.0.0

**Comment by biztos** [comment]
closed, fixed, etc.
